singer/src/main/java/com/pinterest/singer/writer/KafkaWriter.java

```diff
@@ -338,7 +364,25 @@ public void writeLogMessages(List<LogMessage> logMessages) throws LogStreamWrite
           // get the max write latency
           maxKafkaBatchWriteLatency = Math.max(maxKafkaBatchWriteLatency,
               result.getKafkaBatchWriteLatencyInMillis());
+
+          if (SingerSettings.loggingAuditClient != null) {
```

Comment:
Please access variables using getters and setters

Reply:
made the change in update of the PR